src/network/NetworkManagerPrimary.cc

```diff
@@ -27,8 +27,8 @@
 #include <gz/common/Util.hh>
 #include <gz/common/Profiler.hh>
 
-#include "msgs/peer_control.pb.h"
-#include "msgs/simulation_step.pb.h"
+#include "gz/sim/private_msgs/peer_control.pb.h"
+#include "gz/sim/private_msgs/simulation_step.pb.h"
```

Comment:
WorldStatistics, SerializedStateMap, SimulationStep

Reply:
`SerializedStateMap` is in `serialized_map.pb.h`